WebRoot/js/ajax/dwt/xforms/XFormItem.js

```diff
@@ -2323,10 +2328,17 @@ Textarea_XFormItem.prototype.setElementDisabledProperty = function (enable) {
 
 Textarea_XFormItem.prototype.getKeyPressHandlerHTML = function () {
 
-        var keydownEv = "onkeydown";
+        var keydownEv = DwtEvent.ONKEYDOWN; // "onkeydown"
```

Comment:
Can this condition be simplified as above.
ie. by default it will be `var keydownEv = DwtEvent.ONKEYPRESS;`

And in case of IF, Firefox, it will be `DwtEvent.ONKEYDOWN;`

Let me know if you have question.